packages/mgt-element/src/mock/MockGraph.ts

```diff
@@ -74,13 +75,15 @@ class MockMiddleware implements Middleware {
    */
   private _nextMiddleware: Middleware;
 
-  private static _baseUrl;
+  private static _baseUrl: string;
+
+  private static _session: Session = new Session();
 
   // tslint:disable-next-line: completed-docs
   public async execute(context: Context): Promise<void> {
     try {
       const baseUrl = await MockMiddleware.getBaseUrl();
-      context.request = baseUrl + escape(context.request as string);
+      context.request = baseUrl + encodeURI(context.request as string);
```

Comment:
This seems to have broken the agenda story as it uses query string parameters. We're receiving this :  

`{"error":{"code":"ErrorInvalidParameter","message":"This request requires a time window specified by the query string parameters StartDateTime and EndDateTime."}}`

We should ensure that we are encoding the right things on the `?url=` param

Reply:
That should be using `escapeURIComponent`

Reply:
> That should be using `escapeURIComponent`

I think you meant `encodeURIComponent`.

Reply:
I totally did.